_sass/components/_image-list.scss

```diff
@@ -0,0 +1,25 @@
+.image-list {
+  display: grid;
+  grid-template-columns: repeat(1, min-content 1fr);
+  align-items: center;
+  row-gap: 20px;
+
+  .small {
+    font-size: 85%;
+    margin-right: var(--padding-lg);
```

Comment:
**nitpick:** This results in an odd margin on the rightmost column:
![grafik](https://github.com/crystal-lang/crystal-website/assets/466378/8f5456e4-17cb-45f8-8fa8-cb042316dea8)

I think the only way to solve this properly is to merge the icon and text elements into a single grid entry. Then you can use `column-gap` to have a spacing only between columns, but not at the outer edge.